src/workloads/VSSetup.props

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <VSSetupProps>1</VSSetupProps>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <VSDropServiceUri>https://vsdrop.corp.microsoft.com/file/v1/</VSDropServiceUri>
+        <DropServiceUri>https://devdiv.artifacts.visualstudio.com/</DropServiceUri>
+        <DropExe>$(MSBuildThisDirectory)Tools\Drop.App\lib\net45\Drop.exe</DropExe>
+        <!-- Default drop expiration date is 10 years from now -->
+        <DropExpiration Condition="'$(DropExpiration)' == ''">10</DropExpiration>
+        <DropExpirationDate>$([System.DateTime]::Now.AddYears($(DropExpiration)).ToString("M/d/yyyy h:m:s tt"))</DropExpirationDate>
+        <!-- Timeout in minutes -->
+        <DropTimeout>10</DropTimeout>
+        <!-- Can be set to 'info', 'warn', 'error', 'verbose' -->
+        <DropTraceLevel>verbose</DropTraceLevel>
+
+        <!-- Commandline parameters for drop.exe -->
+        <DropParamService>-s &quot;$(DropServiceUri)&quot;</DropParamService>
+        <DropParamTimeout>--timeout &quot;$(DropTimeout)&quot;</DropParamTimeout>
+        <DropParamTraceLevel>--tracelevel &quot;$(DropTraceLevel)&quot;</DropParamTraceLevel>
+        <DropParamExpirationDate>-x &quot;$(DropExpirationDate)&quot;</DropParamExpirationDate>
+        <!-- Use AAD for authentication -->
+        <DropParamAuth>-a</DropParamAuth>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <ManifestTeamProject Condition="'$(ManifestTeamProject)' == ''">dotnet</ManifestTeamProject>
```

Comment:
I think we should add a couple of props here to control the name of the top level .vsman file that is generated, something like:

```xml
    <TargetName>Microsoft.NET.Workload.Runtime</TargetName>
    <ManifestName>$(TargetName)</ManifestName>
```

Reply:
The file vsman rename will work as well, feel free to mark this as resolved.